docs/source/en/quantization/gguf.md

```diff
@@ -77,3 +77,40 @@ Once installed, set `DIFFUSERS_GGUF_CUDA_KERNELS=true`  to use optimized kernels
 - Q5_K
 - Q6_K
 
+## Using Diffusers checkpoints
+
+You can convert a Diffusers checkpoint to GGUF and use it to perform inference. Use the Space below to
+run conversion:
+
+<script
```

Comment:
Maybe try embedding as an iframe instead? The web component currently isn't rendering on preview